transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/wizards/NewTransformationWizard.java

```diff
@@ -73,10 +70,10 @@
 import org.jboss.tools.fuse.transformation.editor.internal.wizards.XMLPage;
 import org.jboss.tools.fuse.transformation.editor.internal.wizards.XformWizardPage;
 import org.jboss.tools.fuse.transformation.extensions.DozerConfigContentTypeDescriber;
-
 import com.sun.codemodel.JCodeModel;
 import com.sun.codemodel.JDefinedClass;
 import com.sun.codemodel.JPackage;
+import javafx.scene.shape.Path;
```

Comment:
Crap, that's happening because of something screwed up in my local env.  I was hoping I could make these changes without it affecting anything, but that obviously is failing.  I'll deal with this tomorrow and update.